File: 프로그래머스/Level1/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;

        int pcnt = 0;
        int ycnt = 0;
        int len = s.length();
        
        for(int i=0; i<len; i++){
            if(s.charAt(i)=='p' || s.charAt(i)=='P'){
                pcnt ++;
            }
            else if(s.charAt(i)=='y' || s.charAt(i)=='Y'){
                ycnt ++;
            }
        }
        
        if(pcnt == ycnt) answer = true;
        else  answer = false;

        return answer;
    }
}

File: 프로그래머스/Level1/문자열 내 p와 y의 개수.js
function solution(s){
    let p = 0, y = 0; 
    
    s = s.toLowerCase();

    for(let i of s){
        if(i === 'p') p++;
        else if(i === 'y') y++; 
    }

    return p === y ? true : false;
}
